### PR TITLE
kirin: manage company_id and test

### PR DESCRIPTION
**Pull request by kadhikari**
For details:  https://jira.kisio.org/browse/NAVP-1031

### CODE REVIEW

File: kirin/cots/model_maker.py [hunk 1]
```diff
@@ -233,6 +234,10 @@ def _make_trip_update(self, vj, json_train):
         trip_message_id = get_value(json_train, 'idMotifInterneReference', nullable=True)
         if trip_message_id:
             trip_update.message = self.message_handler.get_message(index=trip_message_id)
+        cot_company_id = get_value(json_train, 'codeCompagnieTransporteur', nullable=True)
```
Comment (xlqian) [inline kirin/cots/model_maker.py]: cot_company_id = get_value(json_train, 'codeCompagnieTransporteur', nullable=True) or DEFAULT_COMPANY_ID

File: kirin/cots/model_maker.py [hunk 2]
```diff
@@ -371,3 +376,23 @@ def request_navitia_stop_point(self, cr, ci, ch):
...
+    def _get_navitia_company(self, code):
+        """
+        Get a navitia company for the code present in COTS
+        If the company doesn't exist in navitia, another request is made to
+        find company for key="RefProd" and value="1187"
+        """
+        company = self.request_navitia_company(code)
+        if not company:
+            company = self.request_navitia_company(DEFAULT_COMPANY_ID)
+        return company
```
Comment (xlqian) [inline kirin/cots/model_maker.py]:
```python
return self.request_navitia_company(code) or self.request_navitia_company(DEFAULT_COMPANY_ID)
```

File: kirin/cots/model_maker.py (hunk 1 above)
Comment (pbougue) [inline kirin/cots/model_maker.py]: typo: cot`s`_company_id

File: kirin/cots/model_maker.py [hunk 3]
```diff
@@ -371,3 +376,23 @@ def request_navitia_stop_point(self, cr, ci, ch):
...
+
+    def request_navitia_company(self, code):
```
Comment (pbougue) [inline kirin/cots/model_maker.py]: if _get_navitia_company() is "private", so should `_`request_navitia_company() be :wink:

**Pull request merged**